vite.config.ts

```diff
@@ -27,7 +27,7 @@ export default defineConfig(({ mode }) => {
       laravel({
         input: [
           'resources/css/app.css',
-          'resources/js/app.js',
+          'resources/js/index.ts',
```

Comment:
I'd go with `app.ts` as that's the common entry file in Laravel frontends (see `app.css`) and to not confuse it with "simple" index files. Doesn't make that much of a difference to me though.

Reply:
No problem, was wondering about this one. Should be fixed now.